### PR TITLE
Creating custom skip function for group_ldap_link tests.

### PR DESCRIPTION
**Pull request by jasoncarlsen**

### CODE REVIEW

**Comment by jasoncarlsen** [comment]
Clearly I don't understand what isRunningInCE is.  Working to fix this.

**Comment by roidelapluie** [comment]
You have to call it:

isRunningCE()

File: gitlab/resource_gitlab_group_ldap_link_test.go [hunk 1]
```diff
@@ -25,48 +25,57 @@ func TestAccGitlabGroupLdapLink_basic(t *testing.T) {
...
+func testAccGitlabGroupLdapLinkSkipFunc(testCN string, testProvider string, testDataFile string) func() (bool, error) {
+	return func() (bool, error) {
+		if isCE, _ := isRunningInCE(); isCE {
+			return true, nil
+		} else if testCN == "default" || testProvider == "default" {
+			fmt.Printf("[WARNING] Skipping test until test data is configured in %s.\n", testDataFile)
+			return true, nil
+		}
+
+		return false, nil
```
Comment (roidelapluie) [inline gitlab/resource_gitlab_group_ldap_link_test.go:78]:
```suggestion
	return func() (bool, error) {
		if testCN == "default" || testProvider == "default" {
			return true, nil
		}

		return isRunningInCE()
```

**Comment by jasoncarlsen** [comment]
@roidelapluie Thanks!  I searched the code base for isRunningInCE to see what it was and how it worked.  I think my updates should solve the issue.

**Comment by roidelapluie** [comment]
Would you apply my suggestion?

**Comment by jasoncarlsen** [comment]
Updated based on feedback are in.  I have minor reservations about removing the warning statement since these tests are essentially hidden now.  The warning served to inform users that these test exist and can be run by configuring the testdata file.

**Comment by roidelapluie** [comment]
This is at most info.

Do we run those tests in this repo?

**Comment by jasoncarlsen** [comment]
Right now, we don't.  I don't know what ldap provider (if any) is configured in the EE instance this repo runs tests against nor do I know what ldap CNs would be valid.

**Comment by roidelapluie** [comment]
I will see what we can do.

Thanks!

**Pull request merged**